test/action_test.dart

```diff
@@ -201,16 +186,188 @@ void main() {
 
         Completer syncCompleter;
         Completer asyncCompleter;
-        var action = Action();
-        action.listen((_) => syncCompleter.complete());
+        var action = Action()
+          ..listen((_) => syncCompleter.complete())
+          ..listen((_) async {
+            asyncCompleter.complete();
+          });
+        stopwatch.start();
+        for (var i = 0; i < sampleSize; i++) {
+          syncCompleter = Completer();
+          asyncCompleter = Completer();
+          await action();
+          await Future.wait([syncCompleter.future, asyncCompleter.future]);
+        }
+        stopwatch.stop();
+        var averageStreamDispatchTime =
+            stopwatch.elapsedMicroseconds / sampleSize / 1000.0;
+
+        print('awaitable action (ms): $averageActionDispatchTime; '
+            'stream-based action (ms): $averageStreamDispatchTime');
+      }, skip: true);
+    });
+  });
+
+  group('ActionV2', () {
+    ActionV2<String> action;
+
+    setUp(() {
+      action = ActionV2<String>();
+      addTearDown(() async {
+        await action.dispose();
+      });
+    });
+
+    test('should only be equivalent to itself', () {
+      ActionV2 _action = ActionV2();
+      ActionV2 _ActionV2 = ActionV2();
+      expect(_action == _action, isTrue);
+      expect(_action == _ActionV2, isFalse);
```

Comment:
This was probably a find/replace error too.

```suggestion
      ActionV2 action = ActionV2();
      ActionV2 action2 = ActionV2();
      expect(action == action, isTrue);
      expect(action == action2, isFalse);
```